gensim/models/word2vec.py

```diff
@@ -1539,9 +1539,15 @@ def n_similarity(self, ws1, ws2):
           True
 
         """
-        v1 = [self[word] for word in ws1]
-        v2 = [self[word] for word in ws2]
-        return dot(matutils.unitvec(array(v1).mean(axis=0)), matutils.unitvec(array(v2).mean(axis=0)))
+        if len(ws1) > 0 and len(ws2) > 0:
+            v1 = [self[word] for word in ws1]
+            v2 = [self[word] for word in ws2]
+            return dot(matutils.unitvec(array(v1).mean(axis=0)),
```

Comment:
We don't use vertical indent; please use hanging indent (see [PEP8 example](https://www.python.org/dev/peps/pep-0008/#indentation)).